rpcd/playbooks/roles/rpc_maas/defaults/main.yml

```diff
@@ -297,6 +297,8 @@ cloud_resource_warning_vcpus: 80.0
 cloud_resource_critical_vcpus: 90.0
 cloud_resource_warning_disk_space: 80.0
 cloud_resource_critical_disk_space: 90.0
+cloud_resource_cpu_allocation_ratio: 2
+cloud_resource_mem_allocation_ratio: 1
```

Comment:
What happens if these ratios are not the defaults? At a minimum this needs documenting somewhere. Maybe it would be better to set these equal to the ratio vars used by os_nova and then set them in user_variables. This would prevent anyone getting caught out.